lib/job-iteration/active_record_enumerator.rb

```diff
@@ -39,6 +39,12 @@ def size
 
     private
 
+    def instrument_next_batch(cursor)
+      ActiveSupport::Notifications.instrument("cursor.iteration") do
```

Comment:
```suggestion
      ActiveSupport::Notifications.instrument("active_record_cursor.iteration") do
```

Reply:
Hey 👋  @etiennebarrie @Mangara 
Pushed the tweak if you could take another look.